packages/next/bin/next

```diff
@@ -38,6 +38,16 @@ if (inspectArg) {
   nodeArgs.push(inspectArg)
 }
 
+const nodeExtraArgs = process.argv.find(arg => arg.includes('--node-args'))
+
+if (nodeExtraArgs) {
+  const args = nodeExtraArgs.replace(/--node-args=(.+)/, '$1').split(' ')
```

Comment:
I think it's time to implement a proper args parser. Let's use `arg` our super small parser: https://www.npmjs.com/package/arg

After this PR it'll make sense to remove `minimist` and move all args parsing to `arg`